src/SmartEnum/Core/IThen.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Ardalis.SmartEnum.Core
```

Comment:
The namespace can be simply `Ardalis.SmartEnum`.  This saves the users to have to add `using` clauses or `Core` for all interface uses.